ui/apps/platform/src/Containers/VulnMgmt/Dashboard/VulnMgmtDashboardPage.js

```diff
@@ -23,16 +24,21 @@ import MostCommonVulnerabilities from '../widgets/MostCommonVulnerabilities';
 import DeploymentsWithMostSeverePolicyViolations from '../widgets/DeploymentsWithMostSeverePolicyViolations';
 import ClustersWithMostOrchestratorIstioVulnerabilities from '../widgets/ClustersWithMostOrchestratorIstioVulnerabilities';
 
-const entityMenuTypes = [
-    entityTypes.CLUSTER,
-    entityTypes.NAMESPACE,
-    entityTypes.DEPLOYMENT,
-    entityTypes.COMPONENT,
-];
+const baseEntityMenuTypes = [entityTypes.CLUSTER, entityTypes.NAMESPACE, entityTypes.DEPLOYMENT];
+const componentMenuType = [entityTypes.COMPONENT];
+const splitComponentMenuTypes = [entityTypes.NODE_COMPONENT, entityTypes.IMAGE_COMPONENT];
 
 const VulnDashboardPage = ({ history }) => {
     const workflowState = useContext(workflowStateContext);
     const searchState = workflowState.getCurrentSearchState();
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+
+    let entityMenuTypes = [...baseEntityMenuTypes];
+    if (isFeatureFlagEnabled('ROX_FRONTEND_VM_UDPATES')) {
+        entityMenuTypes = [...baseEntityMenuTypes, ...splitComponentMenuTypes];
+    } else {
+        entityMenuTypes = [...baseEntityMenuTypes, ...componentMenuType];
+    }
```

Comment:
Feature flagged the entities we show in the VM dashboard menu dropdown

<img width="153" alt="Screen Shot 2022-06-24 at 4 11 36 PM" src="https://user-images.githubusercontent.com/4805485/175721670-4b97fb63-fd8c-49e1-ac80-dcbb6aadcff5.png">
<img width="148" alt="Screen Shot 2022-06-24 at 4 11 53 PM" src="https://user-images.githubusercontent.com/4805485/175721671-f82cf7b6-2cef-448a-98f9-f2d2f1f34fcd.png">